### ISSUE TITLE
feat - manage no empty constructor in records  

### ISSUE THREAD

**Issue by mxyns** [noahra/java#9]
records don't have an empty constructor (constructor without parameters). this can be worked around by adding an empty constructor in your record and calling this(default parameters) inside of it :
```java
record TestRecord1(long field1) {  
        public TestRecord1() {  
            this(0);  
        }  
    }  
```

need to add a way to handle this by taking the default record constructor and give it default (dummy) values

**Comment by mxyns** [comment]
can't give dummy values, record fields are final and their modification using reflection is deprecated (see comment in #11)

**Comment by mxyns** [comment]
closed in #13
